tests/Gedmo/Sortable/Fixture/ItemWithDateColumn.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/*
+ * This file is part of the Doctrine Behavioral Extensions package.
+ * (c) Gediminas Morkevicius <gediminas.morkevicius@gmail.com> http://www.gediminasm.org
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Gedmo\Tests\Sortable\Fixture;
+
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Mapping\Annotation as Gedmo;
+use Gedmo\Sortable\Entity\Repository\SortableRepository;
+
+/**
+ * @ORM\Entity(repositoryClass="Gedmo\Sortable\Entity\Repository\SortableRepository")
+ */
+#[ORM\Entity(repositoryClass: SortableRepository::class)]
+class ItemWithDateColumn
+{
+    /**
+     * @var int|null
+     *
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column(type: Types::INTEGER)]
+    private $id;
+
+    /**
+     * @Gedmo\SortablePosition
+     * @ORM\Column(type="integer")
+     */
+    #[Gedmo\SortablePosition]
+    #[ORM\Column(type: 'integer')]
+    private $position = 0;
+
+    /**
+     * @Gedmo\SortableGroup
+     * @ORM\Column(type="date")
+     */
+    #[Gedmo\SortableGroup]
+    #[ORM\Column(type: 'date')]
+    private $date;
+
+    /**
+     * @Gedmo\SortableGroup
+     * @ORM\Column(nullable=false)
+     */
```

Comment:
```suggestion
    /**
     * @var string|null
     *
     * @Gedmo\SortableGroup
     * @ORM\Column(nullable=false)
     */
```
Can you please update the getter and setter to use `string|null` instead of `mixed`?